modules/custom/az_flexible_page/config/install/core.entity_view_display.node.az_flexible_page.default.yml

```diff
@@ -35,3 +35,4 @@ hidden:
   field_az_media_image: true
   field_az_page_category: true
   field_az_summary: true
+  smart_title: true
```

Comment:
I think this is correct despite https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_core/config/install/smart_title.settings.yml#L6

Especially considering https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/config/optional/block.block.az_barrio_page_title.yml#L23